### DISCUSSION TITLE
How to get play status in to Home Assistant?

### DISCUSSION THREAD

**Discussion by Joshfindit** [sle118/squeezelite-esp32#188] [Q&A]
What I would like to do is use Home Assistant to turn on the speakers when someone plays music through squeezelite-esp32, and turn them off when they disconnect.

Ideally there is some functionality in squeezelite that can send the notification to HA, but I would also be fine with some type of endpoint that can be polled.

The status should change whether it's LMS, Airplay, Bluetooth, or any other playback method.

Apologies if I'm missing it in the documentation. I searched but did not see anything.

**Comment by jamesmyatt** [comment]
You can integrate the players directly using the slimproto integration: https://www.home-assistant.io/integrations/slimproto/

**Reply by Joshfindit** [reply]
Unfortunately the slimproto integration only sees the play status in either one of two cases:

1. The music is played from HomeAssistant itself
2. The music is played via the squeezelite protocol

It does not show play status for Bluetooth or AirPlay playback

Related: https://github.com/sle118/squeezelite-esp32/discussions/195
